File: jacivan033/atividade_avaliativa1/questao5.py
print("**Seguradora XYZ**")
print(">>Sistema de avaliacao<<")

cs = int(input("Quantidade de contratos simples: "))
cc = int(input("quantidade de contratos completos: "))
cst = 2300 * (cs)/10
cct = 3500 * (cc)/10
lt = int(cst) + int(cct)

print("O lucro obtido foi de: R$ " + str(lt) +",00")

File: jacivan033/atividade_avaliativa1/questao5.java
import java.util.Scanner;
public class questao5 {
        public static void main (String[] args) {
            int s1;
            int c;
            int st;
            int ct;
            int lt;
            Scanner s;
            
            s = new Scanner(System.in);
            System.out.println("**Seguradora CYZ**");
            System.out.println(">>Sistema de avaliaçao<<");

            System.out.println("Quantidade de contratos simples:");
            s1 = s.nextInt();
            System.out.println("Quantidade de contratos completos:");
            c = s.nextInt();

            st = 2300*s1/10;
            ct = 3500*c/10;
            lt = st + ct;

            System.out.println( );
            System.out.println("O lucro obtido foi de: R$ " + lt + ",00");
    }
}
